test/Microsoft.ML.Tests/Scenarios/Api/CookbookSamples/CookbookSamplesDynamicApi.cs

```diff
@@ -171,7 +171,7 @@ private ITransformer TrainOnIris(string irisDataPath)
             var trainedModel = pipeline.Fit(trainData);
 
             // Inspect the model parameters. 
-            var modelParameters = trainedModel.LastTransformer.Model as MulticlassLogisticRegressionModelParameters;
+            var modelParameters = trainedModel.LastTransformer.Model as MaximumEntropyModelParameters;
```

Comment:
>MaximumEntropyModelParameters [](start = 72, length = 29)

obligatory "Update md file" comment

Reply:
Will do now! Thanks.

---
In reply to: [266639456](https://github.com/dotnet/machinelearning/pull/2976#discussion_r266639456) [](ancestors = 266639456)